spec/features/guest_registers_for_courses_spec.rb

```diff
@@ -48,14 +48,19 @@
     expect(page).to have_button("Register for #{date.second.strftime('%B')} Classes")
     expect(page).to have_button("Register for #{date2.second.strftime('%B')} Classes")
     select "Follow", from: "registration_holder[first_role]", match: :first
+    fill_in "Credit Card Number", with: "4242424242424242", match: :first
+    fill_in "card_code", with: "123", match: :first
+    select "1 - January", from: "card_month", match: :first
+    select "2016", from: "card_year", match: :first
     click_button "Register for #{date.second.strftime('%B')} Classes"
+
     expect(page).to have_content("Registration Created!")
     emiline = Student.find_by(first_name: "Emiline")
     expect(emiline.courses.first).to be_a(Course)
     expect(emiline.course_registrations.first).to be_a(CourseRegistration)
   end
 
-  scenario 'guest registers for four classes' do
+  scenario 'guest registers for four classes', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.